src/components/DetailModal/index.js

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+
+const DetailModal = ({data, setData, skills, setSkills, selectedID}) => {
+    return (
+        <>
+            <div className="modal fade" id="jobDetails" tabIndex="-1" aria-labelledby="jobDetailsLabel"
+                 aria-hidden="true">
+                <div className="modal-dialog modal-lg">
+                    <div className="modal-content">
+                        <div className="modal-header">
+                            <p className="modal-title fs-6" id="exampleModalLabel">
+                                <strong>{data.job_title} - {data.company}</strong></p>
```

Comment:
Is the `<strong>` tag actually necessary?

To quote the MDN docs:
> The `<strong>` element is for content that is of "strong importance," including things of great seriousness or urgency (such as warnings).

Is the bold look purely a style thing (in which case Bootstrap's `fw-bold` would do it) or should a screen reader be stressing the title and company to the listener?

Reply:
Changed this with fw-bold